sycl/plugins/unified_runtime/ur/adapters/cuda/device.hpp

```diff
@@ -39,7 +42,19 @@ struct ur_device_handle_t_ {
     UR_CHECK_ERROR(cuDeviceGetAttribute(
         &MaxRegsPerBlock, CU_DEVICE_ATTRIBUTE_MAX_REGISTERS_PER_BLOCK,
         cuDevice));
-  }
+
+    // Set local mem max size if env var is present
+    static const char *LocalMemSizePtr =
```

Comment:
Does it need to be static?

Reply:
I think better for it to be static since then for multiple devices there will only be a single getenv syscall